lib/web/loader/wasm/test/index.test.js

```diff
@@ -195,5 +196,15 @@ DIDKitLoader.loadDIDKit("/didkit_wasm_bg.wasm").then(
 
       if (verify.errors.length > 0) throw verify.errors;
     });
+
+    test("should resolve did:tz", async () => {
+      await resolveDID("did:tz:delphinet:tz1WvvbEGpBXGeTVbLiR6DYBe1izmgiYuZbq", JSON.stringify({}));
+    });
+
+    test("should resolve did:web", async () => {
+      // did:web:vc.transmute.world has an empty verification method which makes the deserialisation fail
+      // did:web:celehner.com doesn't allow CORS
+      await resolveDID("did:web:identity.foundation", JSON.stringify({}));
```

Comment:
WASM tests pass for me. :+1:

`did:web:vc.transmute.world` should resolve since https://github.com/spruceid/ssi/pull/119.
I've updated `did:web:celehner.com` now to allow CORS.
But one is probably enough; `identity.foundation`'s is the simplest.

Reply:
Just for clarification, the resolution for did:tz is successful, but not correct because api.better-call.dev doesn't return anything (and better-call.dev doesn't allow CORS).

You're right, they both work now! I've removed the comments and put the transmute one as it's more complex.